sdk/src/manifest.rs

```diff
@@ -94,8 +94,30 @@ impl Manifest {
         self.claim_generator.as_str()
     }
 
+    /// Returns a MIME content_type for this asset associated with this manifest.
+    pub fn format(&self) -> &str {
+        self.asset().map(|asset| asset.format()).unwrap_or_default()
+    }
+
+    /// Returns the instance identifier.
+    pub fn instance_id(&self) -> &str {
+        self.asset()
+            .map(|asset| asset.instance_id())
+            .unwrap_or_default()
+    }
+
+    /// Returns a user-displayable title for this manifest
+    pub fn title(&self) -> Option<&str> {
+        self.asset().map(|asset| asset.title())
+    }
+
+    /// Returns a tuple with thumbnail format and image bytes or `None`.
+    pub fn thumbnail(&self) -> Option<(&str, &[u8])> {
```

Comment:
Not a blocker for this PR, but I'd like to consider changing this from a tuple to a struct with name and bytes.